deploy/charts/cert-manager/values/deployment.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+type Deployment struct {
+	// Number of replicas
```

Comment:
This documentation is necessarily generic because it is embedded in `CAInjector,Webhook,Global` etc.
But it might be seen as a backward step, because the original documentation strings were context specific. e.g

> Number of cert-manager *cainjector* replicas

But I think think that's fine since the documentation has the full path of the variable in a previous column anyway.

Reply:
If this becomes a hard requirement, we could move the comments to the `defaultValues` struct in `main.go`.